Automattic/wp-calypso
# Newsletters: add "paid newsletter" checkbox at setup step

Another iteration of https://github.com/Automattic/wp-calypso/pull/76022 — please read details from there.

At setup step show checkbox to enable "paid newsletters onboarding" in the launchpad:

<img width="1389" alt="image" src="https://user-images.githubusercontent.com/87168/234905030-1eb436d5-d31c-4659-b621-bf844aa325aa.png">


This PR doesn't _exactly_ match the styles in screenshot yet but it's meant as a prototype to get technical bits going.

## Proposed Changes

- Adds a checkbox at setup step for revealing "paid newsletter" steps at launchpad later on
- Copy isn't final and tiny adjustments in visuals needed.

## Testing Instructions

- At localhost or Calypso live, open http://calypso.localhost:3000/setup/newsletter
- At "setup" step, note that there's a checkbox
- You don't yet see the steps at launchpad; that connection needs more work.



## Pre-merge Checklist
